src/include/buffer.h

```diff
@@ -910,11 +910,6 @@ namespace buffer CEPH_BUFFER_API {
 
     void substr_of(const list& other, unsigned off, unsigned len);
 
-    /// return a pointer to a contiguous extent of the buffer,
-    /// reallocating as needed
-    char *get_contiguous(unsigned off,  ///< offset
```

Comment:
The https://github.com/ceph/ceph/pull/24459#discussion_r223359590 would apply also here.